config/lib/logger.js

```diff
@@ -0,0 +1,46 @@
+﻿'use strict';
+
+var _ = require('lodash'),
+  config = require('../config'),
+  fs = require('fs');
+
+// build logger service
+var logger = {
+  getFormat: getLogFormat, // log format to use
+  getOptions: getLogOptions // log options to use
+};
+
+// export the logger service
+module.exports = logger;
+
+/**
+ * The format to use with the logger
+ *
+ * Returns the log.format option set in the current environment configuration
+ */
+function getLogFormat () {
+  return config.log && config.log.format ? config.log.format : 'combined';
```

Comment:
Why is this defaulting to combined? The current default is 'dev' so we want to change that?